### PR TITLE
translation: router plugin

### PR DESCRIPTION
**Pull request by hanbingleixue**
**What type of PR is this?**

Doc

**What this PR does / why we need it?**

This initiative aims to enhance readability, accessibility, and collaboration for both current and potential contributors, ensuring a unified language throughout the codebase.

**Which issue(s) this PR fixes？**

Fixes #1466

**Does this PR introduce a user-facing change?**

No

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/huaweicloud/Sermant/issues) related with this PR before you start working on it.
- [x] Make sure you have squashed your change to one single commit.
- [x] GitHub Actions works fine in this PR.

### CODE REVIEW

File: sermant-plugins/sermant-router/config/config.yaml [hunk 1]
```diff
@@ -1,22 +1,26 @@
...
+  # Whether to use request information for routing
   use-request-router: false
-  # 使用请求信息做路由时的tags
+  # use the request information as a route tags
   request-tags: []
-  # 需要解析的请求头的tag
+  # the tag of the request header to be resolved
   parse-header-tag: ''
-  # 是否初始化dubbo区域路由，兼容性开关，开启后，会在dubbo的配置缓存中初始化一条以zone匹配的同标签路由
+  # After the compatibility switch is enabled, a route with the same label matching zone will be initialized in the
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/config/config.yaml]: The first sentence is not translated.

File: sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/declarer/AbstractConfigDeclarer.java [hunk 2]
```diff
@@ -19,7 +19,7 @@
 import com.huaweicloud.sermant.core.plugin.agent.matcher.MethodMatcher;
 
 /**
- * 增强AbstractConfig类的appendParameters方法，增加路由标签
+ * The appendParameters method of the Abstract Config class is enhanced to add route labels
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/declarer/AbstractConfigDeclarer.java]: AbstractConfig

File: sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/declarer/AbstractDirectoryDeclarer.java [hunk 3]
```diff
@@ -20,7 +20,8 @@
 import com.huaweicloud.sermant.core.plugin.agent.matcher.MethodMatcher;
 
 /**
- * 增强AbstractDirectory的子类的doList方法，筛选标签应用的地址
+ * The doList method of the Abstract Directory subclass is enhanced to filter the addresses to which the label is
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/declarer/AbstractDirectoryDeclarer.java]: AbstractDirectory

**Review by daizhenyu — DISMISSED** [review]

File: sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/declarer/ApplicationConfigDeclarer.java [hunk 4]
```diff
@@ -19,7 +19,7 @@
 import com.huaweicloud.sermant.core.plugin.agent.matcher.MethodMatcher;
 
 /**
- * 增强ApplicationConfig类的setName方法，用来获取应用名
+ * Enhance the setName method of the Application Configuration class to obtain the application name
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/declarer/ApplicationConfigDeclarer.java]: ApplicationConfiguration

File: sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/interceptor/AbstractConfigInterceptor.java [hunk 5]
```diff
@@ -27,7 +27,7 @@
 import java.util.Map;
 
 /**
- * 增强AbstractConfig类的appendParameters方法，增加路由标签
+ * The appendParameters method of the Abstract Config class is enhanced to add route labels
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/interceptor/AbstractConfigInterceptor.java]: AbstractConfig

File: sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/interceptor/AbstractDirectoryInterceptor.java [hunk 6]
```diff
@@ -23,7 +23,8 @@
 import com.huaweicloud.sermant.router.common.service.AbstractDirectoryService;
 
 /**
- * 增强AbstractDirectory的子类的doList方法，筛选标签应用的地址
+ * The doList method of the Abstract Directory subclass is enhanced to filter the addresses to which the label is
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/interceptor/AbstractDirectoryInterceptor.java]: AbstractDirectory

File: sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/interceptor/ApplicationConfigInterceptor.java [hunk 7]
```diff
@@ -29,7 +29,7 @@
 import java.util.Map;
 
 /**
- * 增强ApplicationConfig类的setName方法，用来获取应用名
+ * Enhance the setName method of the Application Configuration class to obtain the application name
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/interceptor/ApplicationConfigInterceptor.java]: ApplicationConfiguration

File: sermant-plugins/sermant-router/dubbo-router-service/src/main/java/com/huaweicloud/sermant/router/dubbo/strategy/instance/MatchInstanceStrategy.java [hunk 8]
```diff
@@ -22,19 +22,19 @@
 import java.util.function.Function;
 
 /**
- * 匹配目标版本号的invoker
+ * Match the invoice with the target version number
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/dubbo-router-service/src/main/java/com/huaweicloud/sermant/router/dubbo/strategy/instance/MatchInstanceStrategy.java]: invoker，not invoice

File: sermant-plugins/sermant-router/dubbo-router-service/src/main/java/com/huaweicloud/sermant/router/dubbo/strategy/instance/MatchInstanceStrategy.java [hunk 9]
```diff
@@ -22,19 +22,19 @@
...
  *
  * @author provenceee
  * @since 2021-12-08
  */
 public class MatchInstanceStrategy extends AbstractInstanceStrategy<Object, Map<String, String>> {
     /**
-     * 匹配目标版本号的invoker
+     * Match the invoice with the target version number
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/dubbo-router-service/src/main/java/com/huaweicloud/sermant/router/dubbo/strategy/instance/MatchInstanceStrategy.java]: invoker

File: sermant-plugins/sermant-router/dubbo-router-service/src/main/java/com/huaweicloud/sermant/router/dubbo/strategy/type/ObjectTypeStrategy.java [hunk 10]
```diff
@@ -60,11 +60,11 @@ public String getEndFlag() {
     private String getMethodNameByFieldName(String fieldName) {
         char[] chars = fieldName.toCharArray();
         if (Character.getType(chars[0]) == Character.LOWERCASE_LETTER) {
-            // 首字母转大写
+            // Capitalization
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/dubbo-router-service/src/main/java/com/huaweicloud/sermant/router/dubbo/strategy/type/ObjectTypeStrategy.java]: Initial letter to uppercase

File: sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/handler/FlowRouteHandlerTest.java [hunk 11]
```diff
@@ -195,7 +195,7 @@ public void testGetTargetInvokersByRequest() {
         queryMap.put("interface", "com.huaweicloud.foo.FooTest");
         DubboCache.INSTANCE.putApplication("com.huaweicloud.foo.FooTest", "foo");
 
-        // 匹配foo: bar2实例
+        // Matching instances with foo bar2
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/handler/FlowRouteHandlerTest.java]: foo: bar2

File: sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/strategy/RuleStrategyHandlerTest.java [hunk 12]
```diff
@@ -210,7 +212,8 @@ private void setFallbackRoute() {
     }
 
     /**
-     * rule中设置route、fallback，且权重均有命中tag，但是invoker均未命中版本实例的情况
+     * In the rule, both route and fallback are set, and the weights all hit the tag, but the voters all miss the
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/strategy/RuleStrategyHandlerTest.java]: voters?

File: sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/strategy/RuleStrategyHandlerTest.java [hunk 13]
```diff
@@ -221,13 +224,15 @@ public void testApacheV1NotMathRouteFallback() {
...
-        // Route随机命中route1，但是没有0.0.1版本实例，fallback也未命中tag实例，所以返回全部实例信息
+        // Route randomly hits route 1, but there is no instance of version 0.0.1,
+        // and fallback does not hit the tag instance, so all instance information is returned
         List<Object> matchInvokers = RuleStrategyHandler.INSTANCE.getFlowMatchInvokers("foo", invokers, rule);
         Assert.assertEquals(2, matchInvokers.size());
     }
 
     /**
-     * rule中设置routes但权重计算未命中，invoker命中fallback版本实例的情况
+     * The situation where routes are set in the rule but weight calculation misses, and the caller hits the fallback
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/strategy/RuleStrategyHandlerTest.java]: use invoker replace caller

File: sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/strategy/type/ObjectTypeStrategyTest.java [hunk 14]
```diff
@@ -22,14 +22,14 @@
...
- * 实体匹配策略测试
+ * entity matching strategy testing
  *
  * @author provenceee
  * @since 2021-12-01
  */
 public class ObjectTypeStrategyTest {
     /**
-     * 测试实体策略
+     * test your entity strategy
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/strategy/type/ObjectTypeStrategyTest.java]: remove your

File: sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java [hunk 15]
```diff
@@ -148,7 +150,7 @@ private static String buildMethodKey(Class<?> clazz, String methodName, Class<?>
         }
         String className = clazz.getName();
 
-        // 初始化StringBuilder的长度是为了性能
+        // Initialize the length of the String Builder for performance purposes
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java]: StringBuilder

File: sermant-plugins/sermant-router/router-config-common/src/test/java/com/huaweicloud/sermant/router/config/strategy/match/MatchStrategyTest.java [hunk 16]
```diff
@@ -91,41 +91,41 @@ public void before() {
...
-        // 正常情况，区分大小写
+        // Normally, test case-sensitive
         Assert.assertTrue(exact.isMatch(new ArrayList<>(strValues), "foo", true));
 
-        // 测试大小写不一致
+        // Test case inconsistencies
         Assert.assertFalse(exact.isMatch(new ArrayList<>(strValues), "foO", true));
 
-        // 测试null arg
+        // Test null arg and
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/router-config-common/src/test/java/com/huaweicloud/sermant/router/config/strategy/match/MatchStrategyTest.java]: no and

File: sermant-plugins/sermant-router/router-config-common/src/test/java/com/huaweicloud/sermant/router/config/utils/RuleUtilsTest.java [hunk 17]
```diff
@@ -65,7 +65,7 @@ public void testGetTags() {
     }
 
     /**
-     * 测试初始化需要缓存的key
+     * The key that needs to be cached for test initialization
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/router-config-common/src/test/java/com/huaweicloud/sermant/router/config/utils/RuleUtilsTest.java]: test: initialize the key that needs to be cached.

File: sermant-plugins/sermant-router/router-config-common/src/test/java/com/huaweicloud/sermant/router/config/utils/RuleUtilsTest.java [hunk 18]
```diff
@@ -84,7 +84,7 @@ public void testInitMatchKeys() {
...
      */
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/router-config-common/src/test/java/com/huaweicloud/sermant/router/config/utils/RuleUtilsTest.java:88]: test: initialize the key that needs to be cached.

File: sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/handler/AbstractConfigHandler.java [hunk 19]
```diff
@@ -37,17 +37,17 @@ public abstract class AbstractConfigHandler implements AbstractHandler {
...
         this.yaml = new Yaml(new SafeConstructor(null));
     }
 
     /**
-     * 是否需要处理
+     * Whether it needs to be processed
      *
-     * @param key 配置key
-     * @return 是否需要处理
+     * @param key Placement key
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/handler/AbstractConfigHandler.java]: configuration key

File: sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/handler/kind/AbstractKindHandler.java [hunk 20]
```diff
@@ -94,10 +94,10 @@ public void handle(DynamicConfigEvent event, String cacheName) {
     }
 
     /**
-     * 是否需要兼容性处理
+     * Whether compatibility processing is required
      *
-     * @param key 配置key
-     * @return 是否需要处理
+     * @param key Placement key
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/handler/kind/AbstractKindHandler.java]: configuration key

File: sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/service/DubboConfigServiceImpl.java [hunk 21]
```diff
@@ -19,7 +19,7 @@
 import com.huaweicloud.sermant.router.dubbo.service.DubboConfigService;
 
 /**
- * 配置服务
+ * Configure the service
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/service/DubboConfigServiceImpl.java]: configuration service

File: sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/service/ConfigService.java [hunk 22]
```diff
@@ -39,7 +39,7 @@
 import java.util.logging.Logger;
 
 /**
- * 配置服务
+ * Configure the service
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/service/ConfigService.java]: configuration service

File: sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/service/SpringConfigServiceImpl.java [hunk 23]
```diff
@@ -19,7 +19,7 @@
 import com.huaweicloud.sermant.router.spring.service.SpringConfigService;
 
 /**
- * 配置服务
+ * Configure the service
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/service/SpringConfigServiceImpl.java]: configuration service

File: sermant-plugins/sermant-router/router-transmit-plugin/src/main/java/com/huaweicloud/sermant/router/transmit/wrapper/AbstractThreadWrapper.java [hunk 24]
```diff
@@ -97,7 +97,8 @@ public T call() throws Exception {
 
     private void before(Object obj) {
         if (cannotTransmit) {
-            // 当开启普通线程透传，不开启线程池透传时，需要在执行方法之前，删除由InheritableThreadLocal透传的数据
+            // If you enable normal thread pass through but do not enable thread pool pass through,
+            // you need to delete the data that is pass passed by Inheritable Thread Local before executing the method
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/router-transmit-plugin/src/main/java/com/huaweicloud/sermant/router/transmit/wrapper/AbstractThreadWrapper.java]: InheritableThreadLocal

File: sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/handler/AbstractMappingHandler.java [hunk 25]
```diff
@@ -23,32 +23,32 @@
...
- * AbstractHandlerMapping处理器
+ * AbstractHandlerMapping processor
  *
  * @author provenceee
  * @since 2023-02-21
  */
 public abstract class AbstractMappingHandler extends AbstractHandler {
     /**
-     * 配置服务
+     * Configure the service
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/handler/AbstractMappingHandler.java]: configuration service

File: sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/interceptor/EurekaHttpClientInterceptor.java [hunk 26]
```diff
@@ -29,7 +29,7 @@
 import com.netflix.appinfo.InstanceInfo;
 
 /**
- * EurekaHttpClient增强类，eureka注册方法
+ * Eureka Http Client Enhancement Class, Eureka Registration Method
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/interceptor/EurekaHttpClientInterceptor.java]: EurekaHttpClient

File: sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/interceptor/FeignClientInterceptor.java [hunk 27]
```diff
@@ -164,7 +166,8 @@ private Map<String, List<String>> decodeTags(Map<String, List<String>> headers)
 
     private boolean canLoadHystrix() {
         if (PluginConfigManager.getPluginConfig(TransmitConfig.class).isEnabledThreadPool()) {
-            // 开启线程池异步路由时，透传标签不需要依赖HystrixRequestContext，所以直接返回false
+            // When asynchronous routing is enabled for thread pools, transparent transmission tags do not need to rely
+            // on the Hystrix Request Context, so false is returned
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/interceptor/FeignClientInterceptor.java]: HystrixRequestContext

File: sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/service/SpringConfigService.java [hunk 28]
```diff
@@ -21,31 +21,31 @@
 import java.util.Set;
 
 /**
- * 配置服务
+ * Configure the service
```
Comment (daizhenyu) [inline sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/service/SpringConfigService.java]: configuration service

File: sermant-plugins/sermant-router/config/config.yaml (hunk 1 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/config/config.yaml]: resolved

File: sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/declarer/AbstractConfigDeclarer.java (hunk 2 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/declarer/AbstractConfigDeclarer.java]: resolved

File: sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/declarer/AbstractDirectoryDeclarer.java (hunk 3 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/declarer/AbstractDirectoryDeclarer.java]: resolved

File: sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/declarer/ApplicationConfigDeclarer.java (hunk 4 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/declarer/ApplicationConfigDeclarer.java]: resolved

File: sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/interceptor/AbstractConfigInterceptor.java (hunk 5 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/interceptor/AbstractConfigInterceptor.java]: resolved

File: sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/interceptor/AbstractDirectoryInterceptor.java (hunk 6 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/interceptor/AbstractDirectoryInterceptor.java]: resolved

File: sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/interceptor/ApplicationConfigInterceptor.java (hunk 7 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/dubbo-router-plugin/src/main/java/com/huaweicloud/sermant/router/dubbo/interceptor/ApplicationConfigInterceptor.java]: resolved

File: sermant-plugins/sermant-router/dubbo-router-service/src/main/java/com/huaweicloud/sermant/router/dubbo/strategy/instance/MatchInstanceStrategy.java (hunk 8 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/dubbo-router-service/src/main/java/com/huaweicloud/sermant/router/dubbo/strategy/instance/MatchInstanceStrategy.java]: resolved

File: sermant-plugins/sermant-router/dubbo-router-service/src/main/java/com/huaweicloud/sermant/router/dubbo/strategy/instance/MatchInstanceStrategy.java (hunk 9 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/dubbo-router-service/src/main/java/com/huaweicloud/sermant/router/dubbo/strategy/instance/MatchInstanceStrategy.java]: resolved

File: sermant-plugins/sermant-router/dubbo-router-service/src/main/java/com/huaweicloud/sermant/router/dubbo/strategy/type/ObjectTypeStrategy.java (hunk 10 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/dubbo-router-service/src/main/java/com/huaweicloud/sermant/router/dubbo/strategy/type/ObjectTypeStrategy.java]: resolved

File: sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/handler/FlowRouteHandlerTest.java (hunk 11 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/handler/FlowRouteHandlerTest.java]: resolved

File: sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/strategy/RuleStrategyHandlerTest.java (hunk 12 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/strategy/RuleStrategyHandlerTest.java]: resolved

File: sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/strategy/RuleStrategyHandlerTest.java (hunk 13 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/strategy/RuleStrategyHandlerTest.java]: resolved

File: sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/strategy/type/ObjectTypeStrategyTest.java (hunk 14 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/dubbo-router-service/src/test/java/com/huaweicloud/sermant/router/dubbo/strategy/type/ObjectTypeStrategyTest.java]: resolved

File: sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java (hunk 15 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/router-common/src/main/java/com/huaweicloud/sermant/router/common/utils/ReflectUtils.java]: resolved

File: sermant-plugins/sermant-router/router-config-common/src/test/java/com/huaweicloud/sermant/router/config/strategy/match/MatchStrategyTest.java (hunk 16 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/router-config-common/src/test/java/com/huaweicloud/sermant/router/config/strategy/match/MatchStrategyTest.java]: resolved

File: sermant-plugins/sermant-router/router-config-common/src/test/java/com/huaweicloud/sermant/router/config/utils/RuleUtilsTest.java (hunk 17 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/router-config-common/src/test/java/com/huaweicloud/sermant/router/config/utils/RuleUtilsTest.java]: resolved

File: sermant-plugins/sermant-router/router-config-common/src/test/java/com/huaweicloud/sermant/router/config/utils/RuleUtilsTest.java (hunk 18 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/router-config-common/src/test/java/com/huaweicloud/sermant/router/config/utils/RuleUtilsTest.java:88]: resolved

File: sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/handler/AbstractConfigHandler.java (hunk 19 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/handler/AbstractConfigHandler.java]: resolved

File: sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/handler/kind/AbstractKindHandler.java (hunk 20 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/handler/kind/AbstractKindHandler.java]: resolved

File: sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/service/DubboConfigServiceImpl.java (hunk 21 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/service/DubboConfigServiceImpl.java]: resolved

File: sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/service/ConfigService.java (hunk 22 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/service/ConfigService.java]: resolved

File: sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/service/SpringConfigServiceImpl.java (hunk 23 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/router-config-service/src/main/java/com/huaweicloud/sermant/router/config/service/SpringConfigServiceImpl.java]: resolved

File: sermant-plugins/sermant-router/router-transmit-plugin/src/main/java/com/huaweicloud/sermant/router/transmit/wrapper/AbstractThreadWrapper.java (hunk 24 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/router-transmit-plugin/src/main/java/com/huaweicloud/sermant/router/transmit/wrapper/AbstractThreadWrapper.java]: resolved

File: sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/handler/AbstractMappingHandler.java (hunk 25 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/handler/AbstractMappingHandler.java]: resolved

File: sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/interceptor/EurekaHttpClientInterceptor.java (hunk 26 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/interceptor/EurekaHttpClientInterceptor.java]: resolved

File: sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/interceptor/FeignClientInterceptor.java (hunk 27 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/interceptor/FeignClientInterceptor.java]: resolved

File: sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/service/SpringConfigService.java (hunk 28 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/spring-router-plugin/src/main/java/com/huaweicloud/sermant/router/spring/service/SpringConfigService.java]: resolved

File: sermant-plugins/sermant-router/config/config.yaml [hunk 29]
```diff
@@ -1,22 +1,26 @@
...
+  # the tag of the request header to be resolved
   parse-header-tag: ''
-  # 是否初始化dubbo区域路由，兼容性开关，开启后，会在dubbo的配置缓存中初始化一条以zone匹配的同标签路由
+  # Whether to initialize DUBBO zone routing and whether to switch compatibility toggles，After this feature is enabled,
+  # a route with the same label that matches the zone is initialized in the configuration cache of Dubbo
   enabled-dubbo-zone-router: false
-  # 是否初始化spring cloud区域路由，兼容性开关，开启后，会在spring cloud的配置缓存中初始化一条以zone匹配的同标签路由
+  # Whether to initialize the spring cloud area routing and compatibility switch?
+  # Once enabled, a routing with the same label that matches the zone will be initialized in the spring cloud
+  # configuration cache
```
Comment (lilai23) [inline sermant-plugins/sermant-router/config/config.yaml]: Whether to initialize DUBBO zone routing（compatibility switch）
Whether to initialize the spring cloud zone routing （compatibility switch）

File: sermant-plugins/sermant-router/config/config.yaml [hunk 30]
```diff
@@ -1,22 +1,26 @@
 router.plugin:
-  # 是否适配注册插件
+  # whether compatible with the registration plugin
```
Comment (lilai23) [inline sermant-plugins/sermant-router/config/config.yaml]: sermant-springboot-registry plugin

File: sermant-plugins/sermant-router/config/config.yaml [hunk 31]
```diff
@@ -1,22 +1,26 @@
...
+  # configuration cache
   enabled-spring-zone-router: false
   # compatibility router config 1.0, default is false not support
   enabled-previous-rule: false
 transmit.plugin:
-  # 是否在直接new Thread时传递标签
+  # Whether to pass the label on the direct new thread
   enabled-thread: true
-  # 是否在非定时线程池中传递标签
+  # Whether labels are passed in the pool of unscheduled threads
```
Comment (lilai23) [inline sermant-plugins/sermant-router/config/config.yaml]: transmit not pass

Comment (hanbingleixue) [reply sermant-plugins/sermant-router/config/config.yaml]: resolved

File: sermant-plugins/sermant-router/config/config.yaml (hunk 30 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/config/config.yaml]: resolved

File: sermant-plugins/sermant-router/config/config.yaml (hunk 29 above)
Comment (hanbingleixue) [reply sermant-plugins/sermant-router/config/config.yaml]: resolved

**Review by daizhenyu — APPROVED** [review]

**Review by lilai23 — APPROVED** [review]

**Pull request merged**